app/slashcommands/command_expand_collapse.go

```diff
@@ -73,14 +72,14 @@ func setCollapsePreference(a *app.App, args *model.CommandArgs, isCollapse bool)
 	}
 
 	if err := a.Srv().Store.Preference().Save(model.Preferences{pref}); err != nil {
-		return &model.CommandResponse{Text: args.T("api.command_expand_collapse.fail.app_error"), ResponseType: model.CommandResponseTypeEphemeral}
+		return &model.CommandResponse{Text: args.T("api.command_expand_collapse.fail.app_error") + err.Error(), ResponseType: model.CommandResponseTypeEphemeral}
```

Comment:
Cases like these are what I want to really fix. There is an error coming from the store, but we are not propagating that at all. Instead all the user knows is An error occurred while expanding previews.. But _what exactly failed_?. It wouldn't have been possible to know that until now :)